crates/testing/tests/view_sync_task.rs

```diff
@@ -34,7 +34,8 @@ async fn test_view_sync_task() {
         <TestTypes as hotshot_types::traits::node_implementation::NodeType>::Time::new(5),
         hotshot_types::traits::consensus_api::ConsensusSharedApi::public_key(&api),
         hotshot_types::traits::consensus_api::ConsensusSharedApi::private_key(&api),
-    );
+    )
+    .unwrap();
```

Comment:
Do we want to deal with this `unwarp()` safely by taking failed unwrap() into account?